astropy/wcs/wcsapi/fitswcs.py

```diff
@@ -590,26 +656,35 @@ def beta_from_spectralcoord(spectralcoord):
                 def spectralcoord_from_value(value):
                     if isinstance(value, SpectralCoord):
                         return value
-                    return SpectralCoord(value, observer=observer, target=target, **kwargs)
+                    return SpectralCoord(
+                        value, observer=observer, target=target, **kwargs
+                    )
 
                 def value_from_spectralcoord(spectralcoord):
                     # TODO: check target is consistent between WCS and SpectralCoord,
                     # if they are not the transformation doesn't make conceptual sense.
-                    if (observer is None
-                            or spectralcoord.observer is None
-                            or spectralcoord.target is None):
+                    if (
+                        observer is None
+                        or spectralcoord.observer is None
+                        or spectralcoord.target is None
+                    ):
                         if observer is None:
                             msg = 'No observer defined on WCS'
                         elif spectralcoord.observer is None:
                             msg = 'No observer defined on SpectralCoord'
                         else:
                             msg = 'No target defined on SpectralCoord'
-                        warnings.warn(f'{msg}, SpectralCoord '
-                                      'will be converted without any velocity '
-                                      'frame change', AstropyUserWarning)
+                        warnings.warn(
+                            f'{msg}, SpectralCoord '
+                            'will be converted without any velocity '
+                            'frame change',
```

Comment:
```suggestion
                        warnings.warn(
                            f'{msg}, SpectralCoord will be converted without '
                            'any velocity frame change',
```
Is there a way to join/split strings so as to minimize the number of lines?

Reply:
Unfortunately, `black` will not make changes that change the AST, which and changing hard coded strings like this does change that.

Reply:
But see the discussion at https://github.com/astropy/astropy/pull/13970#discussion_r1011722912